backend/entityservice/logger_setup.py

```diff
@@ -0,0 +1,90 @@
+import os
+import logging.config
+import structlog
+
+import yaml
+
+
+def setup_logging(
+    default_path='logging.yaml',
+    default_level='INFO',
+    env_key='LOG_CFG'
+):
+    """
+    Setup logging configuration
+    """
+    path = os.path.dirname(os.path.abspath(__file__)) + '/' + default_path
+    value = os.getenv(env_key, None)
+    if value:
+        path = value
+    if os.path.exists(path):
```

Comment:
Instead of seeing if the path exists, try opening the file and catch the exception.

```python
try:
    f = open(path)
except FileNotFoundError:
    # handle
else:
    with f:
        # read file
````